node_modules/gh-admins/tests/test-admins.js

```diff
@@ -24,87 +24,280 @@ var TestsUtil = require('gh-tests/lib/util');
+                    // Get all global admins in pages, ensuring each page contains just 1 global
+                    // admin, including the initial one and the created one
+                    AdminsTestsUtil.assertGetAllGlobalAdmins(globalAdminClient, {'batchSize': 1}, function(allGlobalAdmins, responses) {
+                        assert.ok(responses.length > 1);
 
-                    // Delete the global admin, ensuring they no longer exist
-                    return AdminsTestsUtil.assertDeleteGlobalAdmin(globalAdminClient, createdGlobalAdmin.id, callback);
+                        // When fetching with size 1, we'll always have one extra request for the
+                        // empty response. So subtract `responses.length` by 1 for the assertion
+                        assert.strictEqual(allGlobalAdmins.length, responses.length - 1);
+                        AdminsTestsUtil.assertGlobalAdmin(_.findWhere(allGlobalAdmins, {'id': globalAdmin.id}), {'exists': true});
+                        AdminsTestsUtil.assertGlobalAdmin(_.findWhere(allGlobalAdmins, {'id': createdGlobalAdmin.id}), {'exists': true});
+                        return callback();
+                    });
                 });
             });
         });
 
         /**
-         * Test that verifies a global admin can delete their own user account
+         * Test that verifies the authorization of getting a list of global admins
          */
-        it('verify a global admin can delete their own user account', function(callback) {
-            TestsUtil.getGlobalAdminRestClient(function(globalAdminClient) {
-                var username = TestsUtil.generateTestUserId();
+        it('verify authorization of getting a list of global admins', function(callback) {
+            // Ensure anonymous user cannot get global admin list
+            return AdminsTestsUtil.assertGetGlobalAdminsFails(anonymousGlobalAdminClient, null, 401, callback);
+        });
+    });
+
+    describe('Me', function() {
+
+        /**
+         * Test that verifies the "me" feed will be anonymous for anonymous users
+         */
+        it('verify the "me" feed will be anonymous for anonymous users', function(callback) {
+            AdminsTestsUtil.assertGetMe(anonymousGlobalAdminClient, function(me) {
+                AdminsTestsUtil.assertMe(me, {'authenticated': false});
+                return callback();
+            });
+        });
+
+        /**
+         * Test that verifies the "me" feed will specify the expected admin's profile information
+         */
+        it('verify the "me" feed will specify the expected admin\'s profile information', function(callback) {
+            var username = TestsUtil.generateTestUserId();
+            AdminsTestsUtil.assertCreateGlobalAdmin(globalAdminClient, username, username, username, function(createdGlobalAdmin, createdGlobalAdminClient) {
```

Comment:
I'm not a fan of re-using the `username` variable for the other parameters. It makes it harder to review.
I'd be fine with doing:
```javascript
AdminsTestsUtil.assertCreateGlobalAdmin(globalAdminClient, username, 'password', 'displayName', function(createdGlobalAdmin, createdGlobalAdminClient) {
```